test/integration/__utils__/withGitIntegration.js

```diff
@@ -81,6 +81,9 @@ export const withGitIntegration =
     // Init repository with initial commit
     await utils.execGit('init')
 
+    // Rename default main branch to master for tests to pass on local environment
+    await utils.execGit(['branch', '-m', 'master'])
+
```

Comment:
Some tests including the new one fail on local env, so I added this rename from `main` to `master`.

Reply:
I don't like that the test branch has to be changed to `master`. I assume it's because _lint-staged_ repo's own default branch is `master` and it somehow leaks into tests.

I wonder if this can be fixed by adding this somewhere:
```
git config --global init.defaultBranch main
```

Reply:
See: https://github.com/okonet/lint-staged/pull/1303/files#r1244710821

Reply:
In the future master branch could be renamed to main, as [Github suggests](https://github.com/github/renaming).